connection.js

```diff
@@ -0,0 +1,15 @@
+// get the client
+const mysql = require('mysql2');
+ 
+// Create the connection pool. The pool-specific settings are the defaults
+const pool = mysql.createPool({
```

Comment:
Can you please hide these credentials using "dotenv" module

Reply:
I have hidden the database credentials